rapid7/metasploit-framework
# Update to the latest meterpreter_bins

This removes checked-in sniffer extension in favor of the gem-packaged version, fixing rapid7/meterpreter_bins#11.

It also pulls in the changes for verifying #4411.
# Steps to Verify
- [x] Run 'bundler install' to get the latest meterpreter_bins gem.
- [x] Start a meterpreter session and ensure that 'load sniffer' works as loaded from the gem .dll file

You can also verify the steps for #4411 and make sure that the things there still work on master without the actual #4411 changes (ensure backward compatibility).
